CONTRIBUTING.md

```diff
@@ -116,13 +120,13 @@ Other environment variables are needed to provide information that allows the pr
 
 ```sh
 # General discussion on GitHub: <https://github.com/EthanThatOneKid/acmcsuf.com/discussions/categories/newsletter>
-VITE_GH_DISCUSSION_CATEGORY_ID=DIC_kwDOE7ysSc4CAC0o
+GH_DISCUSSION_CATEGORY_ID=DIC_kwDOE7ysSc4CAC0o
 
 # Algo discussion on GitHub: <https://github.com/EthanThatOneKid/acmcsuf.com/discussions/categories/algo>
-VITE_GH_ALGO_CATEGORY_ID=DIC_kwDOE7ysSc4COe-m
+GH_ALGO_CATEGORY_ID=DIC_kwDOE7ysSc4COe-m
 
```

Comment:
There is no need to mention `GH_ALGO_CATEGORY_ID`, an unused environment variable.

Reply:
Done.